src/Compilers/VisualBasic/Portable/Syntax/Syntax.xml

```diff
@@ -771,7 +771,7 @@ Unify Enum declarations with other block type declarations.
         <description>The "Inherits" keyword.</description>
       </child>
 
-      <child name="Types" list="true" separator-kind="CommaToken" separator-name="Commas" kind="NonArrayTypeName">
+      <child name="Types" list="true" separator-kind="CommaToken" separator-name="Commas" kind="NonArrayNonTupleTypeName">
```

Comment:
:memo: I changed a few places to use the "NonArrayNonTupleTypeName" I just introduced, because it seems correct, even though they don't affect the generated code.